apps/designer/app/shared/design-system/stitches.config.ts

```diff
@@ -219,6 +219,7 @@ export const {
       7: "45px",
       8: "65px",
       9: "80px",
+      10: "105px",
```

Comment:
please check with @taylornowotny and/or @thysultan regarding the grid we have in the design system

Reply:
Will check the grid and ask if i don't find it

Reply:
According to the design system everything should actually be in increments of 8, I updated it